benchmark-service/benchmark/utils.py

```diff
@@ -45,6 +49,17 @@ def list_benchmark_files(read_gs):
     return list_of_files
 
 
+async def submit_test_batch(batch_client, sha):
+    batch = batch_client.create_batch()
+    job = batch.create_job(image='ubuntu:18.04',
+                           command=['/bin/bash', '-c', 'touch /io/test; sleep 180'],
```

Comment:
Alright. I'm so sorry for changing my mind. But I'm worried that 180 is way too long. Let's do 5. The problem is if there's 10 commits and they're each sleeping that long and we only have 3 cores to share amongst all the test_batch_* and test_ci, we're going to run into timeouts.